services-camel/src/test/resources/solr-update-it-context.xml

```diff
@@ -229,12 +229,17 @@
         <property name="facetFieldFactory" ref="facetFieldFactory" />
     </bean>
     
+    <bean id="solrAccessRestrictionUtil" class="edu.unc.lib.dl.search.solr.util.AccessRestrictionUtil">
+        <property name="searchSettings" ref="searchSettings" />
+        <property name="disablePermissionFiltering" value="true" />
+    </bean>
+    
     <bean id="solrSearchService" class="edu.unc.lib.dl.search.solr.service.SolrSearchService">
         <property name="solrSettings" ref="solrSettings" />
         <property name="searchSettings" ref="searchSettings" />
         <property name="facetFieldUtil" ref="facetFieldUtil" />
         <property name="solrClient" ref="embeddedSolrServer" />
```

Comment:
i'm noticing that this bean has one more property `<property name="solrClient" ref="embeddedSolrServer" />` than the SolrSearchService bean in admin/src/main/webapp/WEB-INF/solr-search-context.xml. why is this?

Reply:
I wouldn't mind simplifying this at some point, but solrClient can either be injected as a property or get initialized in the initializeSolrServer method (now found in `AbstractQueryService`). The solrClient is being provided by another client (`solrSettings`) in an unsuccessful attempt to centralize creation of the solrClient, but I don't think its a good pattern. It wasn't related to this PR though so I was trying to control my refactor urges.